### ISSUE TITLE
Sometimes the icon's css gets modified

### ISSUE THREAD

**Issue by pjvandehaar** [barbushin/javascript-errors-notifier#51]
See <https://www.robinsloan.com/notes/>.

The site's css contains:

```
img {
  width: 100%;
  min-width: 100%;
}
```

To fix this, just add to your css:

```
width: initial !important;
min-width: initial !important;
```

But, to fix future problems like this, does almost every css property need to be set to `initial` for both the icon and the iframe?

**Comment by rbartoli** [comment]
This has been addressed in PR #48. Pending review.

**Comment by barbushin** [comment]
Fixed in 3.0.10. Will be published in a few hours.
Thanks for report, Peter!

**Comment by pjvandehaar** [comment]
Thanks– your fix looks good.

Inevitably, some site is going to do something insane like `img { opacity: 0.1; }` or `img { margin-bottom: 100px; }`.  Shadow DOM could help, but this fix is probably good enough.
